### PR TITLE
Adding info on number of nodes to each element docstring

### PR DESCRIPTION
**Pull request by timryanb**

### CODE REVIEW

File: tacs/elements.pyx [hunk 1]
```diff
@@ -789,9 +846,11 @@ cdef class Quad16NonlinearShell(Element):
     .. seealso:: :ref:`theory/shell_element:Mixed Interpolation of Tensorial Components`
 
     .. note::
-        varsPerNode: 6
+        **varsPerNode**: 6
 
-        outputElement: ``TACS.BEAM_OR_SHELL_ELEMENT``
+        **numNodes**: 16
```
Comment (A-CGray) [inline tacs/elements.pyx:851]: Are these meant to show up in the docs? I couldn't see them when I tried building the docs locally

Comment (timryanb) [reply tacs/elements.pyx:851]: They should show up here: https://smdogroup.github.io/tacs/core/elements.html#tacs.elements.Quad4Shell

Comment (A-CGray) [reply tacs/elements.pyx:851]: I'm not seeing the `numNodes` note you're adding here when I build the docs locally (I'm assuming just checking out this branch without recompiling is all I need to do right?)

image

Comment (timryanb) [reply tacs/elements.pyx:851]: No, you need to recompile cython (make interface) and the docs to update

**Review by A-CGray — APPROVED** [review]

**Pull request merged**